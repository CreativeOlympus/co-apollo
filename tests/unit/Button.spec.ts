import { shallowMount } from '@vue/test-utils'
import Button from '@/lib-components/atoms/button/coa-button.vue'

describe('Button.vue', () => {
  it('Render button with children', () => {
    const button = shallowMount(Button, {
      slots: { default: 'Click Here' },
    })

    expect(button.element).toMatchSnapshot()
  })
})
