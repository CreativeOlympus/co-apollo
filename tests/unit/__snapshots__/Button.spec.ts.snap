// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button.vue Render button with children 1`] = `
<button
  class="coa-btn coa-btn-blue coa-btn-normal coa-btn-base"
>
  
  
  
  Click Here
  
  
  
</button>
`;
